app/src/main/java/chat/rocket/android/chatrooms/adapter/RoomsAdapter.kt

```diff
@@ -71,4 +79,27 @@ class RoomsAdapter(private val listener: (RoomUiModel) -> Unit) :
         const val VIEW_TYPE_HEADER = 2
         const val VIEW_TYPE_LOADING = 3
     }
+
+    private val actionsListener = object : ViewHolder.ActionsListener {
+      override fun isActionsEnabled(): Boolean = enableActions
+      override fun onActionSelected(item: MenuItem, room: RoomUiModel) {
+            room.apply {
+                when (item.itemId) {
+                    R.id.action_favorite_room-> {
+                        presenter.toggleFavoriteChatRoom(this.id,  this.favorite==true)
+                    }
+                    R.id.action_leave_room-> {
+
```

Comment:
Let us know if you are planning to add a function to leave the room here.

Reply:
Yaa, I am working on it.